i18n/update_translations.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
```

Comment:
The problem with this is, that we're basically adding another developer dependency with this.
We'll have the same problem if we require that the user needs to install `nodejs`...
I think the best way would be to use a `javascript` script, which is run with `nashorn`, which should be provided by Java 8.
After a short look on google I found [this](http://www.n-k.de/riding-the-nashorn/) where they write about `jjs test.js`.